autoscale_cloudroast/test_repo/autoscale/fixtures.py

```diff
@@ -214,6 +238,10 @@ def setUpClass(cls):
         cls.lbaas_client = lbaas_client
         cls.rcv3_client = rcv3_client
 
+        cls.lc_image_ref, cls.lc_image_ref_alt = fetch_image_ids(images_client)
```

Comment:
OK, I am beginning to see why this was a pain to update. This is a _lot_ of global mutable state. Is this what *all* cloudcafe code looks like?